### ISSUE TITLE
Much slower install time on node v20

### ISSUE THREAD

**Issue by skutyepov** [uhop/node-re2#197]
I noticed that the time to install re2 on node v20 went up significantly as compared to node v18. It happens on my local intel based OS X machine as well as in Docker with amd64/ubuntu:jammy. 

It takes 7s to install on node v18:
```
node --version
v18.18.2
npm --version
10.2.3
npm install --save re2@1.20.7

added 103 packages, and audited 104 packages in 7s

npm ci

added 103 packages, and audited 104 packages in 2s
```

and 2 minutes on node v20:
```
node --version 
v20.9.0
npm --version
10.2.3
npm i --save re2@1.20.7

added 103 packages, and audited 104 packages in 2m

npm ci

added 103 packages, and audited 104 packages in 2m
```

**Comment by uhop** [comment]
Likely this PR will fix it: https://github.com/uhop/install-artifact-from-github/pull/5 &mdash; coming soon.

**Comment by uhop** [comment]
Published as `1.20.8`. Please test and let me know if it helps in your case.

**Comment by skutyepov** [comment]
It's fixed now. Thank you so much!
